torch/distributed/_tensor/redistribute.py

```diff
@@ -261,6 +262,8 @@ def forward(  # type: ignore[override]
 
         local_tensor = input._local_tensor
         output = redistribute_local_tensor(local_tensor, current_spec, target_spec)
+        if force_wait:
+            output = torch.distributed._functional_collectives.wait_tensor(output)
```

Comment:
is it the case that output may or may not be an ACT, so we can't just call output.wait?  (iirc we added some api to ACT for this purpose)

Reply:
Hmm I just wasn’t sure if wait() would properly no-op if you pass it a plain tensor. But if that should work, I’ll try it and see if anything fails

Reply:
I think this is technically a no-op as `wait_tensor` would check if there's a work associate with the tensor and if there is then call `wait()`. 

But I think it would still be good to have a explicitly check `if isinstance(output, AsyncCollectiveTensor)` then call wait.